sci-geosciences/mapserver/metadata.xml

```diff
@@ -7,10 +7,10 @@
 	</maintainer>
 	<use>
 		<flag name="apache">Enable mapserver to run under apache</flag>
-		<flag name="gdal">Enable <pkg>sci-libs/gdal</pkg> library support</flag>
+		<flag name="gdal">Enable <pkg>sci-libs/gdal</pkg> library support; deprecated (assumed on in future)</flag>
```

Comment:
Is it not possible to use the `restrict` attribute here for the affected versions instead of saying "assumed on in the future"?

Reply:
I didn't want users to think the support was _gone_, but actually, they can read the ebuild if they are not sure. This works fine and is a bit cleaner.